org.ghidra_sre.Ghidra.json

```diff
@@ -78,15 +75,6 @@
                 {
                     "type": "file",
                     "path": "org.ghidra_sre.Ghidra.metainfo.xml"
-                },
-                {
-                    "type": "script",
-                    "dest-filename": "ghidra.sh",
-                    "commands": [
-                        "# Create a default ghidra.properties if one wasn't already set, for the user to edit",
```

Comment:
Your commit subject is far too sparse for this change. You need to explain the impact of the change in the commit subject (I'm guessing it's "Always use upstream launch.properties"?), and explain how that change was implemented in the commit message.

Reply:
Ok

Reply:
I don't understand how that solves the 2x scaling, now that we're not copying the launch.properties file for users to modify. The update TODO reference would be https://github.com/bkerler/ghidra_installer/blob/master/install-ghidra.sh#L24-L31

Reply:
It's not solving the 2x scaling. It's only solving the use of a symbolic link in `/bin`.
So indeed we should replace the link by your.

Reply:
Done